org.kde.qmlkonsole.json

```diff
@@ -0,0 +1,75 @@
+{
+    "id": "org.kde.qmlkonsole",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "command": "qmlkonsole",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.freedesktop.Flatpak"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+        {
+            "name": "kirigami-addons",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/kirigami-addons/kirigami-addons-0.7.0.tar.xz",
+                    "sha256": "0132a58b150229544a61e17e792c48cfc6daa03a8be9c7acd873a6c30d283df7",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 242933,
+                        "stable-only": true,
+                        "url-template": "https://download.kde.org/stable/kirigami-addons/kirigami-addons-$version.tar.xz"
+                    }
+                }
+            ],
+            "config-opts": [
+                "-DBUILD_TESTING=OFF"
+            ]
+        },
+        {
+            "name": "qmltermwidget",
+            "buildsystem": "qmake",
+            "make-install-args": [ "INSTALL_ROOT=/app" ],
+            "post-install": [
+                "mkdir -p /app/lib/qml",
+                "ln -s /app/usr/lib/qml/QMLTermWidget /app/lib/qml",
```

Comment:
I'm pretty sure the problem here is that PREFIX isn't followed.

Reply:
Yes, qmake is by itself not able to install to a different prefix than the one the Qt install uses.
So I've seen this workaround in other flatpaks as well.